src/style.css

```diff
@@ -1 +1,31 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: 96px;
+  height: 20px;
+}
+
+.stars:last-of-type {
+  height: 16px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+}
```

Comment:
```suggestion
.stars__star {
  width: 16px;
  height: 16px;
  background-image: url('./images/star.svg');
  margin-right: 4px;
  background-position: center;
  background-repeat: no-repeat;
}
```

Reply:
Hi
I use this css below, instead of your suggestion. 
With my code we don't need .stars__star:last-of-type  margin-right:0

.stars {
  display: flex;
  justify-content: space-between;
  width: 96px;